apis/python/src/tiledbsoma/io/outgest.py

```diff
@@ -203,14 +202,14 @@ def to_anndata(
     obsm_varm_width_hints: Optional[Dict[str, Dict[str, int]]] = None,
     uns_keys: Optional[Sequence[str]] = None,
 ) -> ad.AnnData:
-    """Converts the experiment group to `AnnData <https://anndata.readthedocs.io/>`_
-    format. Choice of matrix formats is following what we often see in input
-    ``.h5ad`` files:
+    """Converts the experiment group to AnnData format.
+
+    The choice of matrix formats is following what we often see in input ``.h5ad`` files:
 
     * ``X`` as ``scipy.sparse.csr_matrix``
-    * ``obs``,``var`` as ``pandas.dataframe``
-    * ``obsm``,``varm`` arrays as ``numpy.ndarray``
-    * ``obsp``,``varp`` arrays as ``scipy.sparse.csr_matrix``
+    * ``obs``, ``var`` as ``pandas.dataframe``
```

Comment:
Shows visible (and unbalanced) literal backtick characters unless there's a space after the comma.